recipes/dlib/all/conanfile.py

```diff
@@ -33,7 +36,10 @@ class DlibConan(ConanFile):
         "with_png": True,
         "with_sse2": "auto",
         "with_sse4": "auto",
-        "with_avx": "auto"
+        "with_avx": "auto",
+        "with_blas": True,
+        "with_lapack": True,
+        "with_mkl_fft": True
```

Comment:
You need to add these to the requirements as well to avoid picking up system dependencies.
Right now, the build log contains:
```
-- Searching for BLAS and LAPACK
-- Searching for BLAS and LAPACK
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Checking for module 'cblas'
--   No package 'cblas' found
-- Checking for module 'lapack'
--   No package 'lapack' found
```
So currently, these are not detected, and it can potentially detect system libraries.